postcss.config.js

```diff
@@ -1,6 +1,10 @@
+const { join } = require('path')
+
 module.exports = {
   plugins: {
-    tailwindcss: {},
+    tailwindcss: {
+      config: join(__dirname, 'tailwind.config.js'),
```

Comment:
Why was it necessary to change the config?

Reply:
There is apparently a bug in the tailwind version we're using that prevents the `global.css` to work correctly. This was one suggestion that I found that might have contributed to solve the problem. But to be honest, I'm not sure what solved the problem at the end.

Reply:
Very good point! I renamed the variables!